lib/commands/resources-update.ts

```diff
@@ -0,0 +1,34 @@
+export class ResourcesUpdateCommand implements ICommand {
+	public allowedParameters: ICommandParameter[] = [];
+
+	constructor(private $projectData: IProjectData,
+		private $errors: IErrors,
+		private $androidResourcesMigrationService: IAndroidResourcesMigrationService) {
+		this.$projectData.initializeProjectData();
+	}
+
+	public async execute(args: string[]): Promise<void> {
+		await this.$androidResourcesMigrationService.migrate(this.$projectData.getAppResourcesDirectoryPath());
+	}
+
+	public async canExecute(args: string[]): Promise<boolean> {
+		if (!args || args.length === 0) {
+			// Command defaults to migrating the Android App_Resources, unless explicitly specified
+			args = ["android"];
+		}
+
+		for (const platform of args) {
+			if (!this.$androidResourcesMigrationService.canMigrate(platform)) {
+				this.$errors.failWithoutHelp(`The ${platform} does not need to have its resources updated.`);
+			}
+
+			if (this.$androidResourcesMigrationService.hasMigrated(this.$projectData.getAppResourcesDirectoryPath())) {
+				this.$errors.failWithoutHelp("The App_Resources have already been updated for the Android platform.");
+			}
+		}
+
+		return true;
+	}
+}
+
+$injector.registerCommand("resources-update", ResourcesUpdateCommand);
```

Comment:
can you rename the command to `resources|migrate`.  This will allow us to have a command hierarchical structure with [this PR](https://github.com/NativeScript/nativescript-cli/pull/3435), where we can name the commands `resources|generate`.

Reply:
Sure!

Reply:
Done.